managed-services/transactional-email/how-to/bimi.mdx

```diff
@@ -0,0 +1,60 @@
+- [Configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+- A domain name that passed [SPF](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), [DKIM](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), and [DMARC](/managed-services/transactional-email/how-to/add-dmarc-record) authentication checks
+- A logo of your brand saved in SVG format
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status of the Organization in which the actions will be performed, or be an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+
+## How does BIMI work?
+
+Before your brand's logo can be displayed, your email must first pass authentication checks. Checks are usually performed through [SPF (Sender Policy Framework)](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), [DKIM (DomainKeys Identified Mail)](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), and [DMARC (Domain-based Message Authentication, Reporting, and Conformance)](/managed-services/transactional-email/how-to/add-dmarc-record) checks, which are supported by Scaleway. Some email clients also require a Verified Mark Certificate (VMC) to ensure the logo's authenticity.
+
+BIMI is a DNS TXT record that you set up to indicate what brand's logo. The record contains the URL to the brand's logo in SVG (Scalable Vector Graphics) format.
+
+When the email passes all authentication checks and the record is considered valid, your brand's logo will be displayed by the email client next to your emails in the inboxes.
+
+## How to set up a BIMI record
+
+In this tutorial we use [Scaleway Domains and DNS](/network/domains-and-dns/how-to/manage-dns-records) to manage DNS records. You can, however, apply the following steps to any DNS client of your choice. Keep in mind that the field names might vary depending on the client.
+
+1. Click **Domains and DNS** in the **Network** section of the [Scaleway console](https://console.scaleway.com) side menu.
+2. Click the domain you want to manage. This can be an internal or external domain. The domain's **Overview** page displays.
+3. Click the **DNS zones** tab. A list of the DNS zones you have configured within the selected domain displays.
+4. Click **+ Add records** to add new records to your DNS zone. A pop-up displays.
+5. Fill in the required information for the record.
+
+    <Lightbox src="scaleway-bimi-record.webp" alt="" />
+
+    | Field | Value   | Notes |
+    | ------| ------- | ----- |
+    | Type  | TXT           |      |
+    | Name  | `default._bimi.<example.com>`   | At Scaleway your domain name is added automatically, so you need only add `default._bimi`. Other DNS clients might require you to add the entire address.  |
```

Comment:
```suggestion
    | Name  | `default._bimi.<example.com>`   | At Scaleway your domain name is added automatically, so you only need to add `default._bimi`. Other DNS clients might require you to add the entire address.  |
```

Reply:
I know the original sentence is correct but as a user, I'd be confused to see it written like that, but ffti